Sources/SourceControl/Repository.swift

```diff
@@ -42,7 +42,7 @@ public struct RepositorySpecifier: Hashable {
 
     /// Returns the cleaned basename for the specifier.
     public var basename: String {
-        var basename = self.location.description.components(separatedBy: "/").last(where: { !$0.isEmpty }) ?? ""
+        var basename = self.url.pathComponents.dropFirst(1).last(where: { !$0.isEmpty }) ??  ""
```

Comment:
sorry: kill extra space:

```suggestion
        var basename = self.url.pathComponents.dropFirst(1).last(where: { !$0.isEmpty }) ?? ""
```